packages/@react-types/shared/src/dom.d.ts

```diff
@@ -55,12 +56,7 @@ export interface AriaValidationProps {
 
 // A set of common DOM props that are allowed on any component
 // Ensure this is synced with DOMPropNames in filterDOMProps
-export interface DOMProps {
-  /**
-   * The element's unique identifier. See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id).
-   */
-  id?: string
-}
+export interface DOMProps extends Pick<HTMLAttributes<HTMLElement>, 'id'> {}
```

Comment:
apologies, i see now we had it this way so it'd appear in our prop tables in our docs. I'll put it back for you

Reply:
No worries, thanks 😉